.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: false
+language: android
+jdk: oraclejdk8
+
+env:
+  matrix:
+    - ANDROID_TARGET=android-24
+
+android:
+  components:
+    - platform-tools
+    - tools
+    - android-24
+    - build-tools-24.0.1
+    - extra-android-support
+    - extra-android-m2repository
```

Comment:
can you verify that you need these 2 `extra-android-` components? See if travsici builds without them.